deploy_rln_contract.sh

```diff
@@ -14,6 +14,7 @@ if [ ! -d "waku-rlnv2-contract" ]; then
 fi
 
 cd /waku-rlnv2-contract
+git checkout $RLN_CONTRACT_REPO_COMMIT
```

Comment:
Maybe to add a check here to be sure the RLN_CONTRACT_REPO_COMMIT has value and/or check success of the command.